src/index.ts

```diff
@@ -17,6 +17,8 @@ import type {
   PowerState,
 } from './internal/types';
 
+const constants = RNDeviceInfo.getConstants();
```

Comment:
Wouldn't it be better to do it like that: https://github.com/react-native-device-info/react-native-device-info/pull/1509/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R20-R28 so it is lazily evaluated?